hadoop-crypto/src/main/java/com/palantir/hadoop/EncryptedFileSystem.java

```diff
@@ -76,6 +79,29 @@ public FSDataOutputStream create(Path path, FsPermission permission,
         return os;
     }
 
+    @Override
+    public boolean rename(Path src, Path dst) throws IOException {
```

Comment:
same thing could be said for `create` and `open` right? maybe the test should just be call one of the FileSystem methods with a path like `foo//bar` and make sure the keystore gets called with `foo/bar`?